a3p-integration/proposals/d:liquidation-visibility/README.md

```diff
@@ -0,0 +1,88 @@
+# Liquidation Visibility
+This proposal aims to perform a `core-eval` upgrade to `vaultFactory`. The purpose of the upgrade is to 
+improve the visibility of liquidating vaults. For the full spec please see [this issue](https://github.com/Jorge-Lopes/liquidation-visibility/issues/2)
+and for minor additional specs see [this one](https://github.com/Jorge-Lopes/agoric-sdk/issues/36).
+
+## Work Backlog
+* Main workspace => https://github.com/Jorge-Lopes/agoric-sdk
+* PR => https://github.com/Agoric/agoric-sdk/pull/8994
+* Release => https://github.com/Jorge-Lopes/agoric-sdk/releases/tag/liq-visibility-a3p-v0.2
+ 
+## Test Structure
+We follow the usual `a3p` flow where we first perform our upgrade at `EVAL` phase then run our tests in `TEST`
+phase. Notice that we don't run anything under `USE` phase as we don't want the changes we make to create the 
+appropriate test circumstances to effect future builds.
+
+To execute our core-eval we use;
+* [test-liquidation-visibility.js](./test-liquidation-visibility.js)
```

Comment:
this is a confusing name since it's not a test. consider `eval-`

Reply:
Understood, thanks. Will mark as resolved when this is done.

Reply:
Fixed in https://github.com/Agoric/agoric-sdk/pull/8994/commits/8483c67417ce9f4d03716efc0d80dd2476c89bf2